agent/php_nrini.c

```diff
@@ -1730,6 +1730,45 @@ static PHP_INI_MH(nr_unsigned_int_mh) {
   return SUCCESS;
 }
 
+static PHP_INI_MH(nr_log_events_max_samples_stored_mh) {
+  nriniuint_t* p;
+  int val = NR_DEFAULT_LOG_EVENTS_MAX_SAMPLES_STORED;
+
+#ifndef ZTS
+  char* base = (char*)mh_arg2;
+#else
+  char* base = (char*)ts_resource(*((int*)mh_arg2));
+#endif
+
+  p = (nriniuint_t*)(base + (size_t)mh_arg1);
+
+  (void)entry;
+  (void)mh_arg3;
+  NR_UNUSED_TSRMLS;
+
+  /* Anything other than a valid value will result in the
+   * default value of NR_DEFAULT_LOG_EVENTS_MAX_SAMPLES_STORED.
+   */
+
+  p->where = 0;
+
+  if (0 != NEW_VALUE_LEN) {
+    val = (int)strtol(NEW_VALUE, 0, 0);
+    if ((0 >= val) || (NR_MAX_LOG_EVENTS_MAX_SAMPLES_STORED < val)) {
```

Comment:
I believe 0 is a valid value.  This looks like 0 would set the value to the max allowed which might be a surprise to a user.

Reply:
If that's the case, then would the nr_span_events_max_samples_stored_mh also have to be modified to accept 0 values, or are 0 values considered valid there? This block was lifted directly from that mh implementation.

Reply:
Yes, it also needs to be updated, see this ticket for reference: https://github.com/newrelic/newrelic-php-agent/issues/421

Reply:
Nice, I can make both changes here + now unless we don't want to change the span event one yet

Reply:
I'd vote to do both changes if you can.

Reply:
I have removed the changes for #421 for inclusion into a future dedicated PR.